### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by hotzenklotz**
I updated the README to include some more information about the purpose and setup of wk-connect. I also tried to make it more clear how one can import new datasets into wk-connect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,82 @@
 # webknossos-connect
-A [webKnossos](https://github.com/scalableminds/webknossos) compatible data connector written in Python
+A [webKnossos](https://github.com/scalableminds/webknossos) compatible data connector written in Python. 
+
+webKnossos-connect serves as an adapter between the webKnossos data store interface and other alternative data storage servers (e.g BossDB) or static files hosted on Google Cloud Storage (e.g. Neurodata Precomputed)
```
Comment (jstriebel) [inline README.md]:
```suggestion
webKnossos-connect serves as an adapter between the webKnossos data store interface and other alternative data storage servers (e.g BossDB) or static files hosted on Cloud Storage (e.g. Neuroglancer Precomputed)
```

File: README.md [hunk 2]
```diff
@@ -1,22 +1,82 @@
...
+```
+
+#### 3.2 webKnossos UI
+Alternatively, new datasets can be added directly through the webKnossos UI. Configure and import a new datasets from the webKnossos dashboard. (Dashboard -> Datasets -> Upload Dataset -> Add wk-connect Dataset) 
+
+(Read more in the webKnossos docs.)[https://docs.webknossos.org/guides/datasets#uploading-through-the-web-browser]
+
+#### 3.2 Default test datasets
+
+By default, some public datasets are added to webKnossos-connect to get you started when using the Docker image. The initial datasets are hosted on [neurodata.io](https://neurodata.io/ndcloud/#data). For access , create a `.env` file with your Neurodata.io credentials:
```
Comment (jstriebel) [inline README.md]:
```suggestion
By default, some public datasets are added to webKnossos-connect to get you started when using the Docker image. Some initial datasets are hosted on [neurodata.io](https://neurodata.io/ndcloud/#data). For access, create a `.env` file with your Neurodata.io credentials:
```

File: README.md [hunk 3]
```diff
@@ -1,22 +1,82 @@
...
   ```
   INSERT INTO "webknossos"."datastores"("name","url","key","isscratch","isdeleted","isforeign","isconnector")
-  VALUES (E'connect',E'http://localhost:8000',E'secret-key',FALSE,FALSE,FALSE,TRUE);
+  VALUES (E'connect', E'http://localhost:8000', E'secret-key', FALSE, FALSE, FALSE, TRUE);
   ```
-2. To use the initial datasets from [neurodata.io](https://neurodata.io/ndcloud/#data), create a `.env` file:
+### 3. Adding Datasets
+Add and configure datasets to webKnossos-connect to make them available for viewing in webKnossos
+
+#### 3.1 REST API
```
Comment (jstriebel) [inline README.md]: I would propose to change the order of the points:
1. default datasets
2. webknossos UI
3. (if at all) API endpoint. I would actually not publicly recommend to use it. Imo it should be an implementation detail and accessed through the wk UI.

**Comment by hotzenklotz** [comment]
As long as we don't have the wk UI ready for Boss, we should document the REST API. We should coument the REST API anyway for automation reasons etc. (perhaps not in the readme but somewhere more suitable)

**Pull request merged**